### PR TITLE
Utilities/du: Set minimum width when printing sizes

### PR DESCRIPTION
**Pull request by supercomputer7**
This change should help forcing basic alignment either on human readable sizes (or just raw number).

### CODE REVIEW

File: Userland/Utilities/du.cpp [hunk 1]
```diff
@@ -231,15 +231,15 @@ u64 print_space_usage(ByteString const& path, DuOption const& du_option, size_t
...
-        out("{}", human_readable_size(size, AK::HumanReadableBasedOn::Base10));
+        out("{:10s}", human_readable_size(size, AK::HumanReadableBasedOn::Base10));
     } else {
-        out("{}", ceil_div(size, du_option.block_size));
+        out("{:06d}", ceil_div(size, du_option.block_size));
     }
 
     if (du_option.time_type == DuOption::TimeType::NotUsed) {
-        outln("\t{}", path);
+        outln(" {}", path);
```
Comment (nico) [inline Userland/Utilities/du.cpp:242]: `du -h` outputs tab-delimited data on the (non-serenity) systems I tried. Why this change?

Comment (supercomputer7) [reply Userland/Utilities/du.cpp:242]: I thought this will look better actually, but I don't mind reverting this :)
